src/js/jsx/sections/style/Radius.jsx

```diff
@@ -77,7 +77,10 @@ define(function (require, exports, module) {
          * @param {object} props
          */
         _calculateState: function (props) {
-            var layers = props.document.layers.selected.filter(function (layer) {
+            var disabled = props.disabled || props.document.layers.selected.some(function (layer) {
```

Comment:
I think you've got the wrong quantifier here. You want "all selected layers do not have radii" but this is "some selected layer does not have a radii." As long as the control is valid for some layer it should be enabled, and the invalid layers should be filtered out in the handler.